src/Diagnostics.Logger/ETWProvider/DiagnosticsETWProvider.cs

```diff
@@ -151,6 +151,38 @@ public void LogCompilerHostClientWarning(string RequestId, string Source, string
         #endregion
 
         #region Data Provider Events (ID Range : 3000 - 3999)
+        [Event(3000, Level = EventLevel.Informational, Channel = EventChannel.Admin, Message = ETWMessageTemplates.LogDataProviderMessage)]
+        public void LogDataProviderMessage(string RequestId, string Source, string message)
```

Comment:
Rename the "message" argument to be  "Message". Otherwise this will result it one additional column in kusto